.github/workflows/upgrade-dev-deps-main.yml

```diff
@@ -4,7 +4,7 @@ name: upgrade-dev-deps-main
 on:
   workflow_dispatch: {}
   schedule:
-    - cron: 0 18 * * 1
+    - cron: 0 0 * * 1
```

Comment:
nit: can you revert this? i believe that you will need to go into the .projenrc.ts of this package and remove its own `depsUpgradeOptions` that you are now honoring over the default.